src/test/java/com/epam/tc/hw3/TestDataHw3.java

```diff
@@ -0,0 +1,31 @@
+package com.epam.tc.hw3;
+
+import io.github.bonigarcia.wdm.WebDriverManager;
+import org.assertj.core.api.SoftAssertions;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.openqa.selenium.support.ui.WebDriverWait;
+import org.testng.annotations.AfterMethod;
+import org.testng.annotations.BeforeMethod;
+
+public class TestDataHw3 extends Constants {
```

Comment:
I do not see any test data in this class. It will be better rename to the `AbstarctBaseTest` because this class contains some basic logic for all tests and remove `extends Constants`